1 table sketch.sql

```diff
@@ -0,0 +1,21 @@
+/*
+musicrentalDB
+
+MusicRental
+    MusicRentalID int not null PK
+    CustomerFirstName varchar (100) not null, not blank
+    CustomerLastName varchar(100) not null, not blank
+    Instrument varchar(100) not null, not blank
+    Category varchar(100) not null, not blank
+    InstrumentID computed column concat first 3 letters of the instrument and of the category
+    MonthlyRentalFee decimal (6,2), not null, must be greater than 0
+    TotalProfit computed column MonthlyRentalFee * months rented
+    DateRented Date not null, not future date
+    DateReturned not null not future date
+    DateInserted datetime not null, default getdate()
+    
+    multy constraint columns
+        DateReturned mus be after data rented 
+        either DateReturned are both blank or both full
```

Comment:
What did you mean by DateReturned both blank or both full?

Reply:
I fixed this constraint it missed a few words

Reply:
there is no need for this constraint, as Profit is a computed column, and you already have constraints on the columns that it is referencing, in order to ensure proper data